docs/docs-content/clusters/data-center/vmware/permissions.md

```diff
@@ -7,186 +7,231 @@ sidebar_position: 60
+| **Datastore**              | Browse datastore                               |
+| **Host**                   | Configuration: Storage partition configuration |
+| **Network**                | Assign network                                 |
+| **Profile-driven Storage** | View                                           |
+| **Sessions**               | Validate session                               |
+| **Storage Views**          | View                                           |
+| **vSphere Tagging**        | Create vSphere Tag<br />Edit vSphere Tag       |
+
+</TabItem>
+
+<TabItem label="API" value="API">
+
+| **Category**       | **Privileges**                                                                                                                                                                                                                                |
+| ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **API Privileges** | Cns.Searchable<br />Datastore.Browse<br />Host.Config.Storage<br />InventoryService.Tagging.CreateTag<br />InventoryService.Tagging.EditTag<br />Network.Assign<br />Sessions.ValidateSession<br />StorageProfile.View<br />StorageViews.View |
 
 </TabItem>
 
 </Tabs>
 
-:::warning
+### Spectro Root Role Assignments
 
-If the network is a Distributed Port Group under a vSphere Distributed Switch (VDS), _ReadOnly_ access to the VDS
-without “Propagate to children” is required.
+The privileges associated with the _Spectro root role_ must be granted via role assignments on specific vSphere objects
+for either the user or a group containing the user. Review the required role assignments to ensure that your user has
+all required privileges on all required objects.
 
-:::
+| **vSphere Object**     | **Propagation** | **Role**            | **Condition**                                                            |
+| ---------------------- | --------------- | ------------------- | ------------------------------------------------------------------------ |
+| **vCenter Root**       | No              | _Spectro root role_ |                                                                          |
+| **Target Datacenter**  | No              | _Spectro root role_ |                                                                          |
+| **Target Cluster**     | No              | _Spectro root role_ |                                                                          |
+| **Target VM Folder**   | Yes             | _Spectro root role_ |                                                                          |
+| **Distributed Switch** | No              | _Spectro root role_ | If the Target Network is a Distributed Port Group residing beneath a VDS |
```

Comment:
```suggestion
| **Distributed Switch** | No              | _Spectro root role_ | If the Target Network is a Distributed Port Group |
```

Distributed Port Group means it's necessarily a distributed switch.

Reply:
Resolved in eff3548d1103ad25895211ffc3fc1076aabc3de1